src/components/markdown-renderer/replace-components/image/image-frame.tsx

```diff
@@ -23,10 +24,17 @@ export const ImageFrame: React.FC<React.ImgHTMLAttributes<HTMLImageElement>> = (
       .catch(err => console.error(err))
   }, [imageProxyEnabled, src])
 
+  if (src === 'https://') {
+    return (
+      <PlaceholderImageFrame alt={alt} title={title} width={props.width} height={props.height} />
+    )
+  }
```

Comment:
I'm not sure if I'm happy with this block. The purpose of the ImageFrame-component is to show a real image. Maybe the image replacer would be a better place?